Jenkinsfile

```
@@ -163,7 +196,7 @@ el7_component_repos = ""
 leap15_component_repos = ""
 def functional_rpms  = "--exclude openmpi openmpi3 hwloc ndctl " +
                        "ior-hpc-cart-4-daos-0 mpich-autoload-cart-4-daos-0 " +
-                       "romio-tests-cart-4-daos-0 hdf5-mpich2-tests-daos-0 " +
+                       "romio-tests-cart-4-daos-0 hdf5-tests-cart-4-daos-0 " +
```

Looks like a merge conflict resolution defect here.
```suggestion
                       "romio-tests-cart-4-daos-0 hdf5-mpich2-tests-daos-0 " +
```

---

I must have been looking at the wrong locally cached copy of master as now what I see agrees with what you have.